android/app/src/main/kotlin/com/jabook/app/jabook/audio/AudioPlayerServiceBootstrapper.kt

```
@@ -0,0 +1,61 @@
+ * Keeps `AudioPlayerService.onCreate()` as thin orchestration while preserving
+ * exact initialization order and fallback behavior.
+ */
+internal object AudioPlayerServiceBootstrapper {
+    fun initialize(service: AudioPlayerService) {
+        val helper = NotificationHelper(service)
+        service.notificationHelper = helper
+
+        val initialNotification =
+            try {
+                helper.createMinimalNotification()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)